code/datums/mind.dm

```diff
@@ -101,7 +101,7 @@
 			output += "<br><b>Your [A.role_text] objectives:</b>"
 		output += "[A.print_objectives(FALSE)]"
 
-	recipient << browse(output, "window=memory")
+	recipient << browse(russian_to_utf8(output), "window=memory")
```

Comment:
It's better to fix memery at input, not at output. Because that info shows not only in Notes.